ducktape/src/test/scala/io/github/arainko/ducktape/issues/Issue26Spec.scala

```diff
@@ -2,17 +2,17 @@ package io.github.arainko.ducktape.issues
 
 import io.github.arainko.ducktape.*
 
-case class A(anotherCaseClass: A.AnotherCaseClass)
-
-object A {
-  case class AnotherCaseClass(name: String)
-
-  // note how AnotherCaseClass is not referred to as A.AnotherCaseClass
-  case class B(anotherCaseClass: AnotherCaseClass)
-}
-
 // https://github.com/arainko/ducktape/issues/26
 class Issue26Spec extends DucktapeSuite {
+  case class A(anotherCaseClass: A.AnotherCaseClass)
```

Comment:
this needs to go back to toplevel, that bug was very, very specific and even the placement of the case class itself matters